src/mme-app/utils/mmeNasUtils.cpp

```diff
@@ -1800,10 +1804,48 @@ void MmeNasUtils::encode_eps_qos(bearer_qos_t& bearerQos, eps_qos_t& eps_qos)
             eps_qos.len += 4;
 }
 
+void MmeNasUtils::encode_act_ded_br_req_nas_pdu(mme::SessionContext* sessionCtxt,
+		mme::BearerContext* bearerCtxt_p, Secinfo& secContext, struct nasPDU *nas)
+{
+	const uint8_t num_nas_elements = 1;
+	nas->elements = (nas_pdu_elements*) calloc(num_nas_elements,
+	                	sizeof(nas_pdu_elements));
+	nas->elements_len = num_nas_elements;
+	nas->header.message_type = ActivateDedicatedBearerContextRequest;
+	nas->header.security_header_type = IntegrityProtectedCiphered;
+	nas->header.proto_discriminator = EPSMobilityManagementMessages;
+	nas->header.eps_bearer_identity = sessionCtxt->getLinkedBearerId();
+	nas->header.seq_no = secContext.getDownlinkCount();
+	secContext.increment_downlink_count();
+	/* If the procedure was triggered by the transport of user data
+	 * via the control plane, the network shall set the PTI value in
```

Comment:
I thought PTI is useful in case of bearer resource command

Reply:
Yes, we will use the PTI received in the Bearer Resource Allocation Request.